test

```diff
@@ -74,13 +74,18 @@ if [ -n "${vetRes}" ]; then
 	exit 255
 fi
 
-if command -v go-nyet >/dev/null 2>&1; then
-  echo "Checking go-nyet..."
-  nyetRes=$(go-nyet -exitWith 0 $FMT)
-  if [ -n "${nyetRes}" ]; then
-          echo -e "go-nyet checking failed:\n${nyetRes}"
-          exit 255
-  fi
-fi
+echo "Checking govet -shadow..."
+for path in $FMT; do
```

Comment:
The FMT variable had an erroneous *.go addition above which can be removed iirc.